ecosystem.config.cjs

```diff
@@ -12,17 +12,17 @@ module.exports = [
 		args: ['start'],
 
 		// General
-		instances: process.env.PM2_INSTANCES ?? '1',
+		instances: parseInt(process.env.PM2_INSTANCES) || 1,
 		exec_mode: process.env.PM2_EXEC_MODE ?? 'cluster',
 		max_memory_restart: process.env.PM2_MAX_MEMORY_RESTART,
 
 		// Control flow
-		min_uptime: process.env.PM2_MIN_UPTIME,
-		listen_timeout: process.env.PM2_LISTEN_TIMEOUT,
-		kill_timeout: process.env.PM2_KILL_TIMEOUT,
+		min_uptime: parseInt(process.env.PM2_MIN_UPTIME) || undefined,
+		listen_timeout: parseInt(process.env.PM2_LISTEN_TIMEOUT) || undefined,
+		kill_timeout: parseInt(process.env.PM2_KILL_TIMEOUT) || undefined,
 		wait_ready: true,
-		max_restarts: process.env.PM2_MAX_RESTARTS,
-		restart_delay: process.env.PM2_RESTART_DELAY ?? 0,
-		autorestart: process.env.PM2_AUTO_RESTART ?? false,
+		max_restarts: parseInt(process.env.PM2_MAX_RESTARTS) || undefined,
+		restart_delay: parseInt(process.env.PM2_RESTART_DELAY) || 0,
```

Comment:
Do we intentionally fallback to `undefined` when values are `0`?

Reply:
I'm guessing that answer is _no_

Reply:
Ohhhh _whoopsy_ youre right, thanks ... :sweat_smile:

Reply:
I just reverted the number conversion to let pm2 handle those itself. At least this specific bug with the boolean is fixed and pm2 starts :+1: